src/Config.php

```diff
@@ -24,7 +24,8 @@ class Config implements ConfigInterface, ArrayAccess, IteratorAggregate
      * @param mixed $context Raw array of configuration options or path to a
      *                       configuration file or directory containing one or
      *                       more configuration files
-     * @param string $prefix A key under which the loaded config will be nested
+     * @param string|null $prefix A key under which the loaded config will be nested
```

Comment:
The prefix should always be a non-null value if provided. It will only be `null` if omitted. We should add a type hint.

```suggestion
     * @param string|array $prefix A key under which the loaded config will be nested
```